src/app.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+const fs = require('fs');
+const dotenv = require('dotenv');
+const minimist = require('minimist');
+
+dotenv.config();
+
+const command = minimist(process.argv.slice(2))._;
```

Comment:
```suggestion
const [, filePath, dirPath] = minimist(process.argv.slice(2))._;
```
Используй деструктуризацию